tools/capable.py

```diff
@@ -182,7 +182,16 @@ def __getattr__(self, name):
     }
 
     u32 uid = bpf_get_current_uid_gid();
-    struct data_t data = {.tgid = tgid, .pid = pid, .uid = uid, .cap = cap, .audit = audit, .insetid = insetid};
+
+    struct data_t data;
+    __builtin_memset(&data, 0, sizeof(data));
+
+    data.tgid = tgid;
+    data.pid = pid;
+    data.uid = uid;
+    data.cap = cap;
+    data.audit = audit;
+    data.insetid = insetid;
```

Comment:
Yeah, adding a new field may introduce some hole in the struct, that is why the verifier complains.

I think `struct data_t data = {}` should be enough, right ?

Reply:
Hum...
This is a bit strange because the previous code should have lead to unused fields being 0:
https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html

I will investigate, I do not really know what is proper to C, to GCC, to BPF, to BCC and to Clang to give a correct answer about your suggestion.